crew

```diff
@@ -817,7 +817,12 @@ def install
     meta = download
     target_dir = unpack meta
     if meta[:source] == true
-      abort "You don't have a working C compiler. Run `crew install buildessential` to get one and try again.".lightred unless system('gcc --version')
+      abort "You don't have a working C compiler. Execute `crew install buildessential` and try again.".lightred unless system('gcc --version')
+      abort "You don't have a working Clang compiler. Execute `crew install buildessential` and try again.".lightred unless system('clang --version')
```

Comment:
Better to add them to the dependency array automatically on building (if not installed)?

Reply:
Again, if you try to build and buildessential is not installed, you won't get that far.

Reply:
I see what you are saying but that would require a ton of package updates.

Reply:
We can add them to `@deps` array in `expand_dependencies` if build is needed

Reply:
Oh, I get you now.  I didn't understand you meant to modify crew.  That does make sense.

Reply:
@supechicken: Thank you for reviewing.  I was afraid this was going to get lost in the PR queue like the others.  I made your suggested change.